src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/StaffCategoryCaseTest.java

```diff
@@ -428,24 +418,6 @@ void shouldRejectCaseRequestedRole_ForRequester_WrongRequesterJurisdiction_S021(
         });
     }
 
-    @Test
-    void shouldRejectCaseRequestedRole_ForRequester_WrongRoleName_S003() {
```

Comment:
why you removed test case?

Reply:
They have become part of a Parameterized Test, which runs the test with a set of parameters, for each set of parameters it will run the set again.